javaLoginCore/src/main/java/com/hamdan/forzenbook/java/login/core/data/network/LoginService.java

```diff
@@ -11,17 +11,17 @@ public interface LoginService {
     final static String EMAIL = "email";
     final static String CODE = "code";
     final static String LOGIN_GET = "login";
-    final static String LOGIN_POST = "login";
+    final static String LOGIN_POST = "login/";
 
     @GET(LOGIN_GET)
-    public Response<Void> requestValidation(
+    public Call<Void> requestValidation(
             @Query(EMAIL) String email
     );
 
     @FormUrlEncoded
     @POST(LOGIN_POST)
-    public Response<LoginResponse> getToken(
+    public Call<LoginResponse> getToken(
```

Comment:
Why does this need to be a call?

Reply:
because it was refusing to work without it being a call